app/Http/Controllers/Admin/AccountsController.php

```diff
@@ -2,9 +2,12 @@
 
 namespace Pterodactyl\Http\Controllers\Admin;
 
+use Alert;
 use Debugbar;
-use Pterodactyl\Models\User;
+use Hash;
+use Uuid;
 
+use Pterodactyl\Models\User as User;
```

Comment:
You don't need to define this `as User` since it is implied that it is `User` by the namespace. If it was say `Pterodactyl\Models\LongUserNamespaceKindaVague` you could tack on `as User` to make it easier to write and read.